tg_bot/modules/helper_funcs/restrict.py

```diff
@@ -0,0 +1,19 @@
+from tg_bot.modules.sql.restrict_sql import check_if_restricted
+from functools import wraps
+
+
+def restrict(func):
```

Comment:
having thought longer, and it seems more sensible to use a dedicated "restrictable" command handler to restrict stuff - that'll mean less function call overheard from the decorators, no need to do splits, and means the logic of @restrict will be less strange.
By strange, I mean:
- right now, its a decorator so must be added to all restrictables
- it then does a check if it starts with a blocked word - but if its been added to that function then it must start with a possibly blocked word already right?
- Also, your restrict_cmd function has hardcoded that it can only work on runs and afk. but what if either of those modules arent loaded? or if theres a new module with a command you might want to restrict?